lib/jsdom/living/nodes/HTMLElement-impl.js

```diff
@@ -28,6 +29,20 @@ class HTMLElementImpl extends ElementImpl {
     });
   }
 
+  attachShadow(options) {
+    switch (options.mode) {
```

Comment:
This appears to be missing the `delegateFocus` option that the [spec](https://www.w3.org/TR/shadow-dom/#dfn-delegates-focus) allows for to allow focus changes through the shadow root.